core/meta/inc/TClass.h

```diff
@@ -173,7 +176,8 @@ friend class TProtoClass;
    TListOfDataMembers *fData;            //linked list for data members
 
    std::atomic<TListOfEnums*> fEnums;        //linked list for the enums
-   TListOfFunctionTemplates  *fFuncTemplate; //linked list for function templates [Not public until implemented as active list]
+   TListOfFunctionTemplates  *fFuncTemplate; //linked list for function templates
```

Comment:
Note that these changes change the TDataMember's comment. I.e. it's a functionality change.